definitions/tools/pvacbind.cwl

```diff
@@ -44,16 +44,26 @@ inputs:
         type: string[]
         inputBinding:
             position: 4
-    epitope_lengths:
+    epitope_lengths_class_i:
         type: int[]?
         inputBinding:
-            prefix: "-e"
+            prefix: "-e1"
+            itemSeparator: ','
+            separate: false
```

Comment:
With 
```cwl
separate: false
```
an input like
```yaml
epitope_lengths_class_i: [5,6]
```
would become
```bash
-e15,6,7
```
in the command line.  I see that `separate: false` is in the old version, but should we take it out for this (and the class_ii input below it)?  (Repeat this comment for other tools with these inputs, as well.)

Reply:
I updated all the tool CWLs to remove the separate option